docs/design-documents/hal/0004-pin-names-general-guidelines.md

```diff
@@ -35,25 +38,17 @@ To achieve meaningful portability of application code across various Mbed Enable
 
 **Definition of LEDs**
 
-All valid LED definitions should be defined as follows in PinNames.h:
+Only add LEDs that are available in the board. This is an example on how to define LEDs in PinNames.h:
 
     // Px_xx relates to the processor pin connected to the LED
     
-    #define LED1 = Px_xx  // LED1
-    #define LED2 = Px_xx  // LED2  
-    #define LED3 = Px_xx  // LED3  
-    #define LED4 = Px_xx  // LED4  
+    LED1 = Px_xx  // LED1
+    LED2 = Px_xx  // LED2  
```

Comment:
If you remove the #define, you then can't make application with:
#ifdef LED1

Reply:
Good point